src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -71,7 +71,7 @@ public void perform(@NonNull Run<?, ?> run, @NonNull FilePath workspace, @NonNul
         boolean isWindows = !launcher.isUnix();
         // JFROG_BINARY_PATH is set according to the main OS.
         // We should convert JFROG_BINARY_PATH to run on the relevant slave OS if main and slave run on different operating systems.
-        String jfrogBinaryPath = Paths.get(env.get(JFROG_BINARY_PATH), Utils.getJfrogCliBinaryName(isWindows)).toString();
+        String jfrogBinaryPath = Paths.get(env.get(JFROG_BINARY_PATH, ""), Utils.getJfrogCliBinaryName(isWindows)).toString();
```

Comment:
Is this the change that makes the plugin use the `jf` executable that's in the PATH, if the JFROG_BINARY_PATH env var isn not configured?

Reply:
Yes. Without it, the URI is `null/jf`